From 6d0d4282fe40af13378ba2492800d34601e7279c Mon Sep 17 00:00:00 2001
From: Jean Trivelly <jean.trivelly@intel.com>
Date: Tue, 9 Apr 2013 15:58:33 +0200
Subject: wifi: Reset netd LOCAL_MODULE to origin name

BZ: 97757

By default, netd is built. It is listed in
build/target/product/base.mk.
So rename netd.bcm to netd in order to compile it
by default. It is required to have it in the board
to be able to boot. Even if no WiFi module is selected.
In TI case, the compile flag "wl12xx-compat" select the
correct netd.ti instead of netd.

Feature   : CWS.WIFI-Common

Change-Id: If1d4083e4525b8a807c49a8bf5aeebe468349735
Orig-Change-Id: I060b58f42189824bfb94d94c39d572148493a52e
Category: device enablement
Domain: CWS.WIFI-Common
Origin: upstream
Upstream-Candidate: no,proprietary
Signed-off-by: Jean Trivelly <jean.trivelly@intel.com>
---
 Android.mk |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 74320f3..ba0faf1 100644
--- a/Android.mk
+++ b/Android.mk
@@ -58,9 +58,8 @@ else
 ########################################
 
 include $(CLEAR_VARS)
-LOCAL_MODULE:= netd.bcm
+LOCAL_MODULE:= netd
 LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_STEM := netd
 LOCAL_CFLAGS :=
 LOCAL_SRC_FILES:=                                      \
                   BandwidthController.cpp              \
-- 
1.7.4.1

